yowza/yowza/ConfigViewController.swift

```diff
@@ -5,34 +5,49 @@ import HockeySDK
 class ConfigViewController: UIViewController {
     @IBOutlet var environmentControl: UISegmentedControl!
     @IBOutlet var authControl: UISegmentedControl!
-    @IBOutlet var restrictAppUsageControl: UISegmentedControl!
     @IBOutlet var versionLabel: UILabel!
-
+    @IBOutlet var appIdTextField: UITextField!
+    @IBOutlet var appSecretTextField: UITextField!
+    @IBOutlet weak var restrictAppUsageControl: UISwitch!
+    
     static let AuthTypes: [BITAuthenticatorIdentificationType] = [.anonymous, .hockeyAppUser, .hockeyAppEmail, .device, .webAuth]
     static let Environments: [Config.Environment] = [.Training, .Rink]
-    static let RestrictAppUsageOptions: [Bool] = [false, true]
+    
+    let authTypeChanged: Bool = false
```

Comment:
HHHUUUUUH?????  

# Y tho

Reply:
_Lukas used Confusion._

It's super effective!